extensions/dicom-seg/src/components/SegmentationPanel.js

```diff
@@ -0,0 +1,466 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import cornerstoneTools from 'cornerstone-tools';
+import cornerstone from 'cornerstone-core';
+import moment from 'moment';
+
+import { utils } from '@ohif/core';
+import { ScrollableArea, TableList, Icon } from '@ohif/ui';
+
+import {
+  BrushColorSelector,
+  BrushRadius,
+  SegmentationItem,
+  SegmentItem,
+  SegmentationSelect,
+} from './index';
+
+import './SegmentationPanel.css';
+import SegmentationSettings from './SegmentationSettings';
+
+const { studyMetadataManager } = utils;
+
+const segmentationModule = cornerstoneTools.getModule('segmentation');
+const DEFAULT_BRUSH_RADIUS = segmentationModule.getters.radius || 10;
```

Comment:
`segmentationModule.getters` is actually an object full of functions.

This is returning undefined as you should grab the radius from the configuration:

```js
const {configuration} = cornerstoneTools.getModule('segmentation');
const DEFAULT_BRUSH_RADIUS = configuration.radius;
```
```

Reply:
Done, thanks.